src/lxc/cmd/lxc_user_nic.c

```diff
@@ -867,7 +868,8 @@ static char *lxc_secure_rename_in_ns(int pid, char *oldname, char *newname,
 	uid_t ruid, suid, euid;
 	char ifname[IFNAMSIZ];
 	char *string_ret = NULL, *name = NULL;
-	int fd = -1, ifindex = -1, ofd = -1;
+	int fd = -1, ifindex = -1;
+	int ofd;
```

Comment:
I don't mind these changes. The only thing is that our coding style dictates that uninitalized variables are put before initialized variables. So this should be:
```C
        int ofd;
	uid_t ruid, suid, euid;
	char ifname[IFNAMSIZ];
	char *string_ret = NULL, *name = NULL;
	int fd = -1, ifindex = -1;
```